book/src/testnet-validator.md

```diff
@@ -1,187 +1,25 @@
-
-#### Docker users
-
-Those using Docker images can start the processes with:
-
-```bash
-$ docker run \
-	--network host \
-	-v $HOME/.lighthouse:/root/.lighthouse sigp/lighthouse \
-	lighthouse --network MY_NETWORK bn --staking --http-address 0.0.0.0
-```
-
-```bash
-$ docker run \
-	--network host \
-	-v $HOME/.lighthouse:/root/.lighthouse \
-	sigp/lighthouse \
-	lighthouse --network MY_NETWORK vc
-```
-
-### Step 6. Leave Lighthouse running
-
-Leave your beacon node and validator client running and you'll see logs as the
-beacon node stays synced with the network while the validator client produces
-blocks and attestations.
-
-It will take 4-8+ hours for the beacon chain to process and activate your
-validator, however you'll know you're active when the validator client starts
-successfully publishing attestations each epoch:
-
-```
-Dec 03 08:49:40.053 INFO Successfully published attestation      slot: 98, committee_index: 0, head_block: 0xa208…7fd5,
-```
-
-Although you'll produce an attestation each epoch, it's less common to produce a
-block. Watch for the block production logs too:
-
-```
-Dec 03 08:49:36.225 INFO Successfully published block            slot: 98, attestations: 2, deposits: 0, service: block
-```
-
-If you see any `ERRO` (error) logs, please reach out on
-[Discord](https://discord.gg/cyAszAh) or [create an
-issue](https://github.com/sigp/lighthouse/issues/new).
-
-Happy staking!
-
-
-## Custom directories
-
-Users can override the default Lighthouse data directories (`~/.lighthouse/{network}`) using the `--datadir` flag. The custom data directory mirrors the structure of any network specific default directory (e.g. `~/.lighthouse/medalla`).
-
-> Note: Users should specify different custom directories for different networks.
-
-Below is an example flow for importing validator keys, running a beacon node and validator client using a custom data directory `/var/lib/my-custom-dir` for the medalla testnet.
-
-```bash
-lighthouse --network medalla --datadir /var/lib/my-custom-dir account validator import --directory <PATH-TO-LAUNCHPAD-KEYS-DIRECTORY>
-lighthouse --network medalla --datadir /var/lib/my-custom-dir bn --staking
-lighthouse --network medalla --datadir /var/lib/my-custom-dir vc
-```
-The first step creates a `validators` directory under `/var/lib/my-custom-dir` which contains the imported keys and [`validator_definitions.yml`](./validator-management.md).
-After that, we simply run the beacon chain and validator client with the custom dir path.
+[mainnet-validator]: ./mainnet-validator.md
+
+Joining an Eth2 testnet is a great way to get familiar with staking in Phase 0.  All users should
+experiment with a testnet prior to staking mainnet ETH.
+
+To join a testnet, you can follow the [Become an Eth2 Mainnet Validator][mainnet-validator]
+instructions but with a few differences:
+
+1. Use the appropriate Eth2 launchpad website:
+    - [Medalla](https://github.com/goerli/medalla/tree/master/medalla)
```

Comment:
These links should probably point to `medalla.launchpad.ethereum.org` and `pyrmont.launchpad.ethereum.org`. I think it makes sense to put Pyrmont first too.